data/sql/updates/pending_db_world/rev_1668944605510491900.sql

```diff
@@ -0,0 +1,27 @@
+--
+UPDATE `creature` SET `MovementType`=2 WHERE `id1`=17882;
+
+SET @CGUID := (SELECT `guid` FROM `creature` WHERE `id1`=17882);
+DELETE FROM `creature_addon` WHERE `guid`=@CGUID;
+INSERT INTO `creature_addon` VALUES
+(@CGUID,@CGUID*10,0,0,1,0,0,'');
+
+DELETE FROM `waypoint_data` WHERE `id` IN (@CGUID*10);
+INSERT INTO `waypoint_data` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`,`delay`,`move_type`,`action`,`action_chance`,`wpguid`) VALUES
+(@CGUID*10,1,143.12927,24.530138,26.791546,0,0,0,0,100,0),
+(@CGUID*10,2,147.65775,5.377345,26.791546,0,0,0,0,100,0),
+(@CGUID*10,3,137.54929,3.156878,26.791546,NULL,15000,0,@CGUID*10,100,@CGUID),
+(@CGUID*10,4,147.90135,15.377421,26.796476,0,0,0,0,100,0),
+(@CGUID*10,5,151.38458,20.670511,26.857647,NULL,15000,0,@CGUID*10,100,@CGUID),
+(@CGUID*10,6,143.12927,24.530138,26.791546,0,0,0,0,100,0),
+(@CGUID*10,7,141.65916,32.419727,26.831083,NULL,15000,0,@CGUID*10,100,@CGUID),
+(@CGUID*10,8,163.50163,11.745237,26.72663,0,0,0,0,100,0),
+(@CGUID*10,9,168.98875,8.392987,26.64813,NULL,15000,0,@CGUID*10,100,@CGUID), 
+(@CGUID*10,10,147.65775,5.377345,26.791546,0,0,0,0,100,0),
+(@CGUID*10,11,137.54929,3.156878,26.791546,NULL,15000,0,@CGUID*10,100,@CGUID),
+(@CGUID*10,12,147.90135,15.377421,26.796476,0,0,0,0,100,0),
+(@CGUID*10,13,151.38458,20.670511,26.857647,NULL,15000,0,@CGUID*10,100,@CGUID);
+
+DELETE FROM `waypoint_scripts` WHERE `id`=@CGUID*10;
+INSERT INTO `waypoint_scripts` VALUES
+(@CGUID*10,1,1,7,1,0,0,0,0,0,@CGUID);
```

Comment:
```suggestion
(@CGUID*10,2,1,7,0,0,0,0,0,0,@CGUID);
```
@UltraNix Are you going to change it?

Reply:
No? It's okay. 1 as it's emote and 1 as it's a emote state.

Reply:
I meant changing the delay to 2 and datalong to 0. The emote only plays once in the video.